.github/workflows/main.yml

```diff
@@ -51,3 +51,11 @@ jobs:
       run: |
         bazel build //typescript/...
         bazel test //typescript/...
+
+    - name: Building //csharp directory (Linux)
+      if: runner.os == 'Linux'
+      run: bazel build //csharp/... --config=linux
```

Comment:
You shouldn't need `--config=linux` or `--config=macos` here with the changes in .bazelrc. They should be enabled automatically.

Reply:
So there is no need for two different steps, correct?

Reply:
Correct. Both runners should be able to run the same command.